qutip/qip/circuit.py

```diff
@@ -172,15 +173,16 @@ def __init__(self, N, reverse_states=True):
         self.gates = []
         self.U_list = []
 
-    def add_gate(self, name, targets=None, controls=None, arg_value=None,
+    def add_gate(self, gate, targets=None, controls=None, arg_value=None,
                  arg_label=None):
         """
         Adds a gate with specified parameters to the circuit.
 
         Parameters
         ----------
-        name: String
-            Gate name.
+        gate: String
```

Comment:
maybe here you could say
```
    String or `Gate`
```
Not sure what the standard on this kind of thing is, but it is what I do